api/v1alpha1/drclusterconfig_types.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// DRClusterConfigSpec defines the desired state of DRClusterConfig
+type DRClusterConfigSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Foo is an example field of DRClusterConfig. Edit drclusterconfig_types.go to remove/update
+	Foo string `json:"foo,omitempty"`
+}
+
+// DRClusterConfigStatus defines the observed state of DRClusterConfig
+type DRClusterConfigStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+}
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+//+kubebuilder:resource:scope=Cluster
+
+// DRClusterConfig is the Schema for the drclusterconfigs API
+//
+//nolint:maligned
```

Comment:
is this line required?

Reply:
As status is empty at present, the linter does not like the structure alignment. `nolint` added till there are elements in status, at which point this can be removed.